includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -342,53 +322,6 @@ public static function sync_reader_from_order( $order, $verify_created_via = tru
 		return \Newspack_Newsletters_Subscription::add_contact( $contact );
 	}
 
-	/**
-	 * If the site is using woocommerce-memberships, create a new user and a
-	 * membership, if the donation type calls for it.
-	 *
-	 * @param string $email_address Email address.
-	 * @param string $full_name Full name.
-	 * @param string $frequency Donation frequency.
-	 * @param array  $metadata Donor metadata.
-	 */
-	public static function set_up_membership( $email_address, $full_name, $frequency, $metadata = [] ) {
```

Comment:
This method [is used in Blocks](https://github.com/Automattic/newspack-blocks/blob/31da96848de290f1db08f4a2940b60b4453fd25d/src/blocks/donate/class-wp-rest-newspack-donate-controller.php#L163-L170), why is it removed here?